feature/system/src/main/java/io/github/droidkaigi/confsched2019/system/actioncreator/ActivityActionCreator.kt

```diff
@@ -60,6 +62,16 @@ class ActivityActionCreator @Inject constructor(val activity: FragmentActivity)
         activity.startActivity(intent)
     }
 
+    fun copyText(label: String, text: String): Boolean {
+        val cpManager =
+            ContextCompat.getSystemService(activity, ClipboardManager::class.java) ?: return false
```

Comment:
I think you can use `ShowProcessingMessage` instead of returning Boolean. What do you think? 👀 

`dispatcher.launchAndDispatch(Action.ShowProcessingMessage(Message.of("...")))`

Reply:
Messages we want to show depend on the situation, so let me create new action for copy text.